scripts/contacts/migrateContactsV1toV2.js

```diff
@@ -0,0 +1,167 @@
+const has = require('lodash/has')
+
+const mkAPI = require('../api')
+const {
+  migrationDiff: diff,
+  getWithInstanceLogger
+} = require('../../libs/utils')
+
+const CONTACTS_APP_NAME = 'Contacts'
+const KONNECTOR_APP_NAME = 'konnector-google'
+const VCARD_APP_NAME = 'vcard'
+const ACCOUNT_APP_VERSION = 1
+const DOCTYPE_CONTACTS = 'io.cozy.contacts'
+const DOCTYPE_ACCOUNTS = 'io.cozy.accounts'
+const DOCTYPE_CONTACTS_ACCOUNT = 'io.cozy.contacts.accounts'
+const DOCTYPE_CONTACTS_VERSION = 2
+
+let client, api
+
+const getCreatedByApp = contact => {
+  const metadataKeys = Object.keys(contact.metadata).filter(
+    k => k !== 'version'
+  )
+
+  if (metadataKeys.includes('google')) {
+    return KONNECTOR_APP_NAME
+  } else if (metadataKeys.includes('cozy')) {
+    return CONTACTS_APP_NAME
+  } else if (metadataKeys.length === 0) {
+    return VCARD_APP_NAME
```

Comment:
The VCard import is part of cozy-contacts though, so not sure we need to distinguish that.

Reply:
ok, I see. I changed it for this: if it's not google, it's Contacts